metrics/web/test/common/presentation/widgets/metrics_user_menu_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/widgets/base_popup.dart';
+import 'package:metrics/common/presentation/routes/observers/overlay_entry_route_observer.dart';
+import 'package:metrics/common/presentation/widgets/metrics_user_menu.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+void main() {
+  group('MetricsUserMenu', () {
+    testWidgets(
+      'retrieves an OverlayEntryRouteObserver from the context and applies it to the BasePopup',
```

Comment:
Would love to see more tests here: what about shows metrics user menu card on tap? Displays as a user avatar image?